src/services/__tests__/helpers.spec.ts

```diff
@@ -167,4 +179,148 @@ describe('helpers', () => {
+          images: [{ id: 2, image: 'image' }],
+          alternatives: [
+            {
+              word: 'Abhänger',
+              article: ARTICLES[2],
+            },
+          ],
+        },
+        {
+          id: 2,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Akkuschrauber',
+          article: ARTICLES[1],
+          audio: '',
+          images: [{ id: 2, image: 'image' }],
+          alternatives: [],
+        },
+        {
+          id: 3,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Auto',
+          article: ARTICLES[3],
+          images: [{ id: 1, image: 'image' }],
+          audio: '',
+          alternatives: [],
+        },
+        {
+          id: 4,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Helm',
+          article: ARTICLES[1],
+          audio: '',
+          images: [{ id: 2, image: 'image' }],
+          alternatives: [],
+        },
+        {
+          id: 5,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Hose',
+          article: ARTICLES[2],
+          audio: '',
+          images: [{ id: 1, image: 'image' }],
+          alternatives: [],
+        },
+        {
+          id: 6,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Oberarm',
+          article: ARTICLES[1],
+          audio: '',
+          images: [{ id: 2, image: 'image' }],
+          alternatives: [],
+        },
+        {
+          id: 7,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Ölkanne',
+          article: ARTICLES[1],
+          audio: '',
+          images: [{ id: 2, image: 'image' }],
+          alternatives: [
+            {
+              word: 'Ölkännchen',
+              article: ARTICLES[3],
+            },
+          ],
+        },
+        {
+          id: 8,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Riffeldübel',
+          article: ARTICLES[1],
+          audio: '',
+          images: [{ id: 2, image: 'image' }],
+          alternatives: [
+            {
+              word: 'Holzdübel',
+              article: ARTICLES[1],
+            },
+          ],
+        },
+
+        {
+          id: 9,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Spachtel',
+          article: ARTICLES[1],
+          images: [{ id: 1, image: 'image' }],
+          audio: 'https://example.com/my-audio',
+          alternatives: [
+            {
+              word: 'Spachtel',
+              article: ARTICLES[2],
+            },
+            {
+              word: 'Alternative',
+              article: ARTICLES[2],
+            },
+          ],
+        },
+        {
+          id: 10,
+          type: VOCABULARY_ITEM_TYPES.lunesStandard,
+          word: 'Untergrund',
+          article: ARTICLES[1],
+          audio: '',
+          images: [{ id: 2, image: 'image' }],
+          alternatives: [],
+        },
+      ]
```

Comment:
Can't you just get those from the VocabularyItemBuilder?

Reply:
@steffenkleinle I wanted to compare this data with the one which was created by the **VocabularyItemBuilder**

Reply:
But basically you are just checking that `getSortedAndFilteredVocabularyItems` doesn't change the input, right? Or am I missing something?

Reply:
@steffenkleinle I am checking the correct order of the list whether the umlaut characters are not at the very end of the list

Reply:
But you are doing
``` 
      const sortedTestData = getSortedAndFilteredVocabularyItems(sortedData, '')
      expect(sortedData).toEqual(sortedTestData)
```
which checks for equality of `sortedData` and `sortedTestData`. So actually you are just making sure that `sortedData` is still the same after calling `getSortedAndFilteredVocabularyItems` on it. Am I missing something?

Reply:
@steffenkleinle you are right. I am comparing unsorted list with sorted

Reply:
However, the naming does not represent this, both is called "sorted", not "unsorted". And if one is sorted and one is unsorted, it should not be equal after sorting, should it?

Reply:
You should pass an unsorted list as params though then, not the already sorted list

Reply:
@steffenkleinle you are right. Now I understand what I did here is a illogical. Thank you:)

Reply:
Good to merge :rocket: